pyaerocom/helpers.py

```diff
@@ -234,10 +233,8 @@ def make_dummy_cube_latlon(
         dummy cube in input resolution
     """
 
-    if lat_range is None:
-        lat_range = (-90, 90)
-    if lon_range is None:
-        lon_range = (-180, 180)
+    # Accept lists for lat_range and lon_range, but make sure correct length
+    assert len(lat_range) == 2 and len(lon_range) == 2
```

Comment:
you can use Python's syntax to simplify this line
``` python
    assert len(lat_range) == len(lon_range) == 2
```
(as long as you find it more readable)

Reply:
Yes, this seems like a good change